adapter.go

```diff
@@ -63,7 +63,7 @@ func (a *Adapter) loadPolicy(ctx context.Context, model model.Model, handler fun
 
 	// Parse the rules from Redis
 	for _, rule := range rules {
-		handler(strings.Split(rule, ","), model)
+		handler(strings.Split(rule, ", "), model)
```

Comment:
"test0,test1,  test2,     test3" is valid syntax, right?
I would suggest splitting by ",", then loop through the slice and trim the spaces.

Sorry can't commit myself right now.

Reply:
> "test0,test1, test2, test3" is valid syntax, right? Your solution wouldn't work for that. I would suggest splitting by ",", then loop through the slice and trim the spaces with strings.TrimSpace().
> 
> Sorry can't commit myself right now.

Forget it, just checked it a bit more and it should be ok.

Reply:
when you save the policy to redis by the rules below, you may split it by ', '
~~~go
func buildRuleStr(ptype string, rule []string) string {
	return ptype + ", " + util.ArrayToString(rule)
}
~~~

Reply:
Yep, exactly what I found out after checking the code for more than 1 minute. ;) Sorry!